pkgs/development/python-modules/tensorboard/default.nix

```diff
@@ -24,15 +24,14 @@
 
 buildPythonPackage rec {
   pname = "tensorboard";
-  version = "2.11.0";
+  version = "2.13.0";
   format = "wheel";
-  disabled = pythonOlder "3.6" || pythonAtLeast "3.11";
```

Comment:
@dav-dap this line was changed in https://github.com/NixOS/nixpkgs/pull/242660 , and now this PR has merge conflicts.